pkg/daemon/node.go

```diff
@@ -43,7 +43,7 @@ func (dn *Daemon) loadNodeAnnotations(node *corev1.Node) (*corev1.Node, error) {
 	}
 
 	glog.Infof("Setting initial node config: %s", initial[constants.CurrentMachineConfigAnnotationKey])
-	n, err := setNodeAnnotations(dn.kubeClient.CoreV1().Nodes(), dn.nodeLister, node.Name, initial)
+	n, err := dn.setNodeAnnotations(initial)
```

Comment:
hm, do you know why for some instances we don't use the retry version of setNodeAnnotations (e.g. here)?

Reply:
Yep, for the critical path (from `syncNode` to reboot) we don't really want to retry, it's the controller mechanism which will retry for us. For things like setting the SSH annotation we don't really care, I could have _not_ add a retry as well as it's retried anyway, but it looked more consistent there to retry 🤷‍♂️

Reply:
Yep, for the critical path (from `syncNode` to reboot) we don't really want to retry, it's the controller mechanism which will retry for us. For things like setting the SSH annotation we don't really care, I could have _not_ add a retry as well as it's retried anyway, but it looked more consistent there to retry 🤷‍♂️